R/asis-nodes.R

```diff
@@ -260,3 +260,38 @@ protect_tickbox <- function(body, ns) {
   }
   copy_xml(body)
 }
+
+# FOOTNOTES --------------------------------------------------------------------
+#nocov start
```

Comment:
why no cov?

Reply:
It was an idea I had and wanted to implement, but I realized it got tricky as trying to keep `[^` and `]` from being escaped might also accidentally escape user-escaped brackets, which look exactly the same to commonmark:

``` r
xml2::read_xml(commonmark::markdown_xml("[ and \\["))
#> {xml_document}
#> <document xmlns="http://commonmark.org/xml/1.0">
#> [1] <paragraph>\n  <text xml:space="preserve">[ and [</text>\n</paragraph>
```

<sup>Created on 2021-05-25 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>